pkg/tsdb/elasticsearch/client/client.go

```diff
@@ -313,7 +340,7 @@ func (c *baseClientImpl) createMultiSearchRequests(searchRequests []*SearchReque
 }
 
 func (c *baseClientImpl) getMultiSearchQueryParameters() string {
-	if c.version >= 70 {
+	if c.version.Major() >= 70 {
```

Comment:
Shouldn't it be 7, not 70?

```suggestion
	if c.version.Major() >= 7 {
```

Reply:
ABSOLUTELY yes. :D 

thanks for catching it!